packages/devtools_app/lib/src/charts/flame_chart.dart

```diff
@@ -384,6 +381,10 @@ abstract class FlameChartState<T extends FlameChart, V> extends State<T>
             FlameChart.maxZoomLevel,
           );
           await zoomTo(newZoomLevel, jump: true);
+          if (newZoomLevel == FlameChart.minZoomLevel &&
+              horizontalController.offset != 0.0) {
```

Comment:
What if there is so much content that a scroll offset other than zero is still valid at the min zoom level?

Reply:
That is not currently how the flame_chart is implemented. At a zoom level of 1.0 (min zoom level), all of the content is rendered in view, so 0.0 is the only scroll offset we would want.